packages/oui-navbar/_mixins.less

```diff
@@ -67,6 +67,15 @@
         color: inherit;
       }
     }
+
+    // Header/Title
+    &__title {
+      display: inline-block;
+      margin-left: rem-calc(15);
+      font-size: rem-calc(28);
+      font-weight: 300;
+      color: #fff;
```

Comment:
I would have used `white` instead 😄

Reply:
Shorted =! clear

```javascript
setTimeout(blastOff, 86400000);
```

is shorter than

```javascript
const MILLISECONDS_IN_A_DAY = 86400000;

setTimeout(blastOff, MILLISECONDS_IN_A_DAY);
```

but not clearer !

But it's only a detail, I don't really care

Reply:
Nevermind, most people seem to agree that `#fff` is better than `white` 😄